app/src/main/java/org/woheller69/weather/weather_api/open_meteo/OMHttpRequest.java

```diff
@@ -25,7 +25,7 @@ protected String getUrlForQueryingOMweatherAPI(Context context, float lat, float
 
         if (sharedPreferences.getBoolean("pref_snow", false)){
             return String.format(
-                    "%sforecast?latitude=%s&longitude=%s&forecast_days=%s&hourly=%s,relative_humidity_2m,rain,showers,snowfall,weather_code,pressure_msl,wind_speed_10m,wind_direction_10m&minutely_15=%s,rain,showers,snowfall,weather_code,wind_speed_10m&forecast_minutely_15=60&daily=weather_code,%s,%s,sunrise,sunset,uv_index_max,rain_sum,showers_sum,snowfall_sum,wind_speed_10m_max,sunshine_duration&current_weather=true&wind_speed_unit=ms&timeformat=unixtime&timezone=auto",
+                    "%sforecast?latitude=%s&longitude=%s&forecast_days=%s&hourly=%s,uv_index,relative_humidity_2m,rain,showers,snowfall,weather_code,pressure_msl,wind_speed_10m,wind_direction_10m&minutely_15=%s,rain,showers,snowfall,weather_code,wind_speed_10m&forecast_minutely_15=60&daily=weather_code,%s,%s,sunrise,sunset,uv_index_max,rain_sum,showers_sum,snowfall_sum,wind_speed_10m_max,sunshine_duration&current_weather=true&wind_speed_unit=ms&timeformat=unixtime&timezone=auto",
```

Comment:
This could be in a more API sensitive way, via only adding "uv_index" if the option is actually selected.
Shall I do that?

Reply:
we can keep it for now. I might change that for pressure as well in future